src/table/utils/handle-accessibility.ts

```diff
@@ -145,3 +145,7 @@ export const announceSelectionState = (
       : announce({ keys: ['SNTable.SelectionLabel.NotSelectedValue'] });
   }
 };
+
+export const handleKeyToCopyCell = (cell: Cell) => {
+  !navigator.clipboard.writeText(cell.qText as string) && alert('The content is not copied!');
```

Comment:
Not sure alert is what we want to have here but thought is good to be for failed cases.

Reply:
writeText is asynchronous, needs to be an await before it and if we want to catch the error we need a try catch block. Should look something like this

```suggestion
  try {
    await navigator.clipboard.writeText(cell.qText as string);
  } catch (error) {
    alert(error);
  }
```

you also need to add `async` to the function definition.

It is a good idea to have some kind of indication of what is going on and that we could potentially get an error back. But we don't to do it with alert, it is not that big of a deal. Not sure what to do here. potentially just log the error in the consol

Reply:
or the other way around, indicate that we have successfully copied to clipboard. But that is not in the old implementation, so no need to do that now